src/ImageSharp/Formats/Jpeg/JpegMetadata.cs

```diff
@@ -23,12 +46,26 @@ private JpegMetadata(JpegMetadata other)
         {
             this.Quality = other.Quality;
             this.ColorType = other.ColorType;
+
+            this.lumaQuantizationTable = other.lumaQuantizationTable;
+            this.chromaQuantizationTable = other.chromaQuantizationTable;
+            this.LumaQuality = other.LumaQuality;
+            this.ChromaQuality = other.ChromaQuality;
         }
 
         /// <summary>
         /// Gets or sets the encoded quality.
         /// </summary>
-        public int Quality { get; set; } = 75;
+        public int Quality
+        {
+            get => (int)Math.Round(this.LumaQuality);
```

Comment:
Current implementation assumes quality is based on luminance channel only, while this would work just fine as luma is more important than chroma, maybe we should calculate this some other way?